src/halmos/config.py

```diff
@@ -274,6 +274,12 @@ class Config:
         action=ParseErrorCodes,
     )
 
+    inv_depth: int = arg(
```

Comment:
nit: I like full names better, can we call this `invariant_depth`?

Reply:
done: https://github.com/a16z/halmos/pull/454/commits/5c6356b250564a5b2c15a1a132d7a8964187121c